### ISSUE TITLE
Install failed on rpi3

### ISSUE THREAD

**Issue by narduin** [YunoHost-Apps/wireguard_ynh#26]
Hello,

I tried to install the app on my raspberry pi 3 but it fails.
Most of the errors seem to come from dependencies "not going to be installed" or "not installable".

### Context

- Hardware: *raspberry pi 3B*
- YunoHost version: 4.2.6.1
- I have access to my server: *Through SSH | through the webadmin | direct access via keyboard / screen*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*

### Logs

[full logs here](https://paste.yunohost.org/raw/otedefihom)

Thanks for helping!

**Comment by tituspijean** [comment]
Oops yes! I discovered the issue this weekend but forgot the send a patch in the main branches.

**Comment by tituspijean** [comment]
@narduin the automated tests are failing right now, but I do confirm it works on my RPi4. Can you try it out on yours?

`sudo yunohost app install https://github.com/YunoHost-Apps/wireguard_ynh/tree/testing`

**Comment by narduin** [comment]
I will hapily test it but I've never installed a testing branch before. Will I have to remove it to install the main when it's patched?

**Comment by narduin** [comment]
I was able to install the app.

However it does not currently work. Both `wg-quick@wg0` and `wireguard_ui` were stopped.
I manually started `wg-quick@wg0` but `wireguard_ui` throws an error:
```
Jun 30 09:17:21 systemd[1]: Started WireGuard UI.
Jun 30 09:17:21 systemd[1879]: wireguard_ui.service: Changing to the requested working directory failed: Permission denied
Jun 30 09:17:21 systemd[1879]: wireguard_ui.service: Failed at step CHDIR spawning /opt/yunohost/wireguard/wireguard-ui: Permission denied
Jun 30 09:17:21 systemd[1]: wireguard_ui.service: Main process exited, code=exited, status=200/CHDIR
Jun 30 09:17:21 systemd[1]: wireguard_ui.service: Failed with result 'exit-code'.
```

**Comment by tituspijean** [comment]
@narduin, can you share the output of `ls -la /opt/yunohost/wireguard/`?

**Comment by tituspijean** [comment]
Issue found and fixed in #27. Feel free to reopen if needed.

**Comment by narduin** [comment]
Latest version installed without problem! 
Thanks 👍 

I can't get it to work but I think it's because the rpi is behind a router with its own firewall/dhcp/etc. ^^
